porting/halium_7-1/Configuring.rst

```diff
@@ -118,6 +118,214 @@ At the end of the file, append this::
+#.  Run the script to integrate the newer sources into your kernel source tree.
+#.  Fix errors that *will* occur when the script is run.
+#.  Add a security patch.
+#.  Make necessary changes to your kernel defconfig file.
+#.  Rebuild and flash the full halium-boot.img.
+#.  Add bluetooth initializaton script, rebuild and flash system.img.
+
+.. _BT-driver:
+
+Determine driver and bluetooth settings for your device
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By the time you reach this point in the porting process, you will have completed building halium-boot (probably a number of times). It is wise to note all kernel defconfig settings related to bluetooth before proceeding with the steps below. One of these settings designates the bluetooth driver used by your device, which you will need to know at a later stage.
+
+The directions below may help you locate the settings in question and their dependencies:
+
+After completing a build of halium-boot.img, go to your ``out/target/product/[device]/obj/KERN_OBJ`` directory. There, run the following command::
+
+    ARCH=arm64 make menuconfig
+
+If your device is armhf, use ``ARCH=arm`` instead.
+
+This will bring up menuconfig complete with the defconfig settings from your build. You then navigate to the bluetooth drivers submenu and browse through all activated settings, recording which ``CONFIG_xxxxx`` settings apply to those that are activated for your device, as well as the information about what this setting does and which other settings it depends upon (found under Help). Save this information for later reference.
+
+Download the backports scripts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Clone the backports scripts into a directory outside your halium source tree by issuing this command from your home (~) directory::
+
+    git clone https://github.com/ubuntu-phonedations/backports.git -b for-ubuntu backport-scripts
+
+This downloads the backports scripts prepared by Canonical based on the :ref:`Backports Project <Backports>` mentioned above, and places them in the directory ``~/backport-scripts``. The scripts are specifically written to backport from kernel version 4.2.
+
+Download kernel source to backport from
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Create a directory (outside your halium source tree) for the kernel source from which you will pull the newer drivers::
+
+    mkdir ~/kernel-backports
+
+Now clone the kernel source for v4.2::
+
+    cd ~/kernel-backports
+    git clone https://kernel.googlesource.com/pub/scm/linux/kernel/git/next/linux-next -b v4.2
+
+.. Note::
+
+    Although there are other kernel versions besides v4.2 available (as witnessed by available version tags on `the webpage <https://kernel.googlesource.com/pub/scm/linux/kernel/git/next/linux-next>`_), the backports script is specifically tailored to backporting from version 4.2 and thus effectively limits you to this option.
+
```

Comment:
I find the 'as witnessed by' a bit overly verbose, well yeah there are other versions. It's not an outlandish claim. Maybe just cut it to 

Although there are `other kernel <LINK>` versions besides v4.2 available, the .... 

And actually since we're just indicated that there are other linux kernel versions out there, I see no reason to point specifically at linux-next. I'd suggest to link to https://www.kernel.org/

Also, I think we should move this whole Note up to the end of the intro. Just before

# Bluetooth backporting steps

Reply:
Will do